Taudem5PCVS2010/FlowdirConditioning/stdafx.h

```diff
@@ -0,0 +1,15 @@
+// stdafx.h : include file for standard system include files,
```

Comment:
This file appears to have been automatically generated. I prefer not to have it. See if we can configure project without it.